internal/server/kong/ws/config/compat/compatibility_table.go

```diff
@@ -636,6 +637,7 @@ var (
 							{
 								Field:          "access",
 								ValueFromField: "functions",
+								IgnoreEmpty:    true,
```

Comment:
`IgnoreEmpty` in this case doesn't make sense; this should be when the `Field` is **not** empty. Another way of putting this is when `config.access` has a value (e.g. not empty) then the value should not be copied and the field update process for this change should be short circuited.

Reply:
I can update it, but we talked about ignoring fields with empty values, not wholesale ignoring existing fields when the target field exists already.

Reply:
Fair point and the described task should have better reflected this. Looking at the issue more closely we wouldn't want to have configured fields (e.g. not empty) be overwritten regardless of what another field has in it configuration.

For example using the `pre-function` serverless plugin, if the plugin is configured the following way:

```json
{
  "access": [
    "kong.log.err('access')"
  ]
  "functions": [
    "kong.log.err('functions')"
  ]
}
```

In this example we would not want to override `config.access` just because `config.functions` is not empty; whatever is assigned is `config.access` in this instance should remain. Does that clear things up?

Reply:
Yes, thank you. I can think of a few of edge cases, like what should the expected behavior be in this case:
```
{
  "access": []
  "functions": [
    "kong.log.err('functions')"
  ]
}
```
I don't think we can answer that generally for all present and future plugin configs.

Reply:
Nope and that is why the flag will be beneficial to have in this case. For this case though if `config.access` is "empty" we want `config.functions` to be copied into `config.access` for >= 3.0.